joint_trajectory_controller/test/test_trajectory_actions.cpp

```diff
@@ -373,145 +396,84 @@ TEST_F(TestTrajectoryActions, test_goal_tolerances_success) {
 
     JointTrajectoryPoint point2;
     point2.time_from_start = rclcpp::Duration::from_seconds(0.3);
-    point2.positions.resize(joint_names.size());
+    point2.positions.resize(joint_names_.size());
 
     point2.positions[0] = 7.0;
     point2.positions[1] = 8.0;
     point2.positions[2] = 9.0;
     points.push_back(point2);
 
-    sendActionGoal(points, 1.0, opt);
+    sendActionGoal(points, 1.0, goal_options_);
   }
-  controller_hw_thread.join();
+  controller_hw_thread_.join();
 
   EXPECT_EQ(true, feedback_recv);
-  EXPECT_EQ(true, common_goal_accepted);
-  EXPECT_EQ(rclcpp_action::ResultCode::SUCCEEDED, common_resultcode);
+  EXPECT_EQ(true, common_goal_accepted_);
+  EXPECT_EQ(rclcpp_action::ResultCode::SUCCEEDED, common_resultcode_);
   EXPECT_EQ(
     control_msgs::action::FollowJointTrajectory_Result::SUCCESSFUL,
-    common_action_result_code);
+    common_action_result_code_);
 
-  EXPECT_NEAR(7.0, test_robot->pos1, common_threshold);
-  EXPECT_NEAR(8.0, test_robot->pos2, common_threshold);
-  EXPECT_NEAR(9.0, test_robot->pos3, common_threshold);
-
-  executor.cancel();
+  EXPECT_NEAR(7.0, test_robot_->pos1, COMMON_THRESHOLD);
+  EXPECT_NEAR(8.0, test_robot_->pos2, COMMON_THRESHOLD);
+  EXPECT_NEAR(9.0, test_robot_->pos3, COMMON_THRESHOLD);
 }
 
 TEST_F(TestTrajectoryActions, test_state_tolerances_fail) {
   // set joint tolerance parameters
   double state_tol = 0.0001;
```

Comment:
need more const

Reply:
also, can this be also a common threshold thing? I think it's fine to test with `0.001`  rad threshold instead of `0.0001` rad

Reply:
Done and done.